macros/dune/ref.sql

```diff
@@ -0,0 +1,9 @@
+{% macro ref(model_name) %}
```

Comment:
Why is this one needed?

Reply:
it's so that wizards who compile locally (without the db connection and with the dummy profile in the repo) don't have the database names in the compiled SQL. This allows the SQL to be run on dune.com without modifications.

Reply:
Aha, where does this show up? I thought we had fixed this in https://github.com/duneanalytics/spellbook/pull/3592

Reply:
Makes sense!